### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by gagalago**
fix tests that are failing by improving their setup phase.

- allow to run them on a free tenant (skip custom domain tests in that case)
- set or not the `username` on a new user depending on the connection configuration (seems to not allow it by default on new tenant)
- add time in some resources (role, user name to be able to run test even if the resource is already present (maybe a better cleanup is better but don't know how to do it so that it run even if tests fail)
- reduce some number to be compliant with default maximum allowed values

### CODE REVIEW

File: management/example_test.go [hunk 1]
```diff
@@ -1,175 +0,0 @@
-package management_test
```
Comment (alexkappa) [inline management/example_test.go:1]: Is this deleted intentionally?

**Review by alexkappa — CHANGES_REQUESTED** [review]
Hi @gagalago, I appreciate the improvements you propose. I'm afraid some of these changes will require changes to the tenant used to run acceptance tests and will likely fail if run.

I'd be reluctant to merge this PR

File: management/example_test.go (hunk 1 above)
Comment (gagalago) [reply management/example_test.go:1]: > Hi @gagalago, I appreciate the improvements you propose. I'm afraid some of these changes will require changes to the tenant used to run acceptance tests and will likely fail if run.
> 
> I'd be reluctant to merge this PR

in fact, for me, tests in the actual situation failed even on a new tenant. that's why I proposed these changes to be run on a new tenant. I added in the pull request description what/why I did.

What are the changes that will be needed on the tenant?

**Comment by gagalago** [comment]
> Hi @gagalago, I appreciate the improvements you propose. I'm afraid some of these changes will require changes to the tenant used to run acceptance tests and will likely fail if run.
> 
> I'd be reluctant to merge this PR

in fact, for me, tests in the actual situation failed even on a new tenant. that's why I proposed these changes to be run on a new tenant. I added in the pull request description what/why I did.

What are the changes that will be needed on the tenant?

**Comment by sergiughf** [comment]
Thanks a lot for opening this PR @gagalago. Completely agree with you that some of the tests are not runnable on a free tenant and the entire setup could be further improved for reliability purposes. 

Unfortunately in its current state this PR is not exactly mergeable as Alex described above and it also deviates from our intent of refactoring and improving these tests by introducing wiremock. We've captured all the information in a JIRA ticket and will work on this in an upcoming version.

**Pull request closed without merging**